### DISCUSSION TITLE
Configuração do Rider

### DISCUSSION THREAD

**Discussion by Lehzinho** [balta-io/forum#329] [Tira Dúvidas] [answered]
Olá! 

Estou enfrentando muitas dificuldades com o JetBrains Rider e gostaria de obter ajuda com a configuração. Quando seleciono um trecho de texto no Rider, não consigo usar funções como 'Ctrl+C' para copiar o texto selecionado ou mesmo digitar qualquer coisa. Além disso, percebi que quando há um trecho selecionado, aparece uma barrinha mais grossa no cursor, como se eu tivesse pressionado a tecla 'Insert'. Isso está afetando minhas atividades de edição e não consigo encontrar uma solução. Alguém sabe o que pode estar causando esse problema e como posso corrigi-lo? Agradeço antecipadamente por qualquer ajuda!

**Comment by andrebaltieri** [comment, marked as answer]
@Lehzinho provavelmente seu KeyMap está diferente do padrão, nas configurações, procure por KeyMap e em seguida selecione a que mais te agrada.

Eu utilizo a nativa do Rider, mas muita gente usa o KeyMap do Visual Studio, vale um teste em ambas para ver qual você gosta mais.

**Reply by Lehzinho** [reply]
Desculpa, André, mas a dica que você deu não adiantou muito, não. Eu tô usando o KeyMap do VSCode e o problema continua do mesmo jeito. Dá uma olhada nessa screenshot que eu vou mandar pra você entender melhor. Valeu!
![Rider](https://github.com/balta-io/forum/assets/99737712/b4f763c4-cb0d-4204-8d59-0b4d1273365b)

**Reply by Lehzinho** [reply]
O problema ocorre quando eu seleciono qualquer texto e, em seguida, nenhuma função do teclado ou KeyMaps funciona. Para sair dessa situação, eu tenho que pressionar as teclas A ou O, mas não consigo realizar nenhuma outra ação quando isso acontece. É isso que está acontecendo.

**Reply by Lehzinho** [reply]
Resolvido por @AndreyMateos valeu desabilitamos a extenção do Vim 
Obrigado

**Reply by AndreyMateus** [reply]
@Lehzinho  Só trocou o "u" pelo "o", mas tá suave, XD.
